code/modules/mob/living/carbon/update_icons.dm

```diff
@@ -244,7 +238,12 @@
 	var/list/new_limbs = list()
 	for(var/X in bodyparts)
 		var/obj/item/bodypart/BP = X
-		new_limbs += BP.get_limb_icon()
+
+		if(istaur && (istype(BP, /obj/item/bodypart/r_leg) || istype(BP, /obj/item/bodypart/l_leg)))
+			continue
+		BP.update_limb()
+		new_limbs += BP.get_limb_icon()
```

Comment:
this used to be
 `var/image/temp = BP.get_limb_icon()
  		if(temp)
  			new_limbs += temp`